3.1/exports/store.md

```diff
@@ -49,3 +49,6 @@ public function storeExcel()
     Excel::store(new InvoicesExport(2018), 'invoices.xlsx', 's3', null, 'private');
 }
 ```
+
+## PS:
+- storing a file is affected by queues, so make sure to have a look @ [Queued Docs](https://docs.laravel-excel.com/3.1/exports/queued.html)
```

Comment:
I would change this to 

If your exports using the `ShouldQueue concern` don't forget to run the queue worker.

Reply:
The way it's written now, it sounds like every stored export has to do with the queue, which is not the case.

Reply:
not exactly, but it means that if u have a queue setup for that export, it doesnt need `->queue()` for it work, using `->store()` is also affected by it.

Reply:
You only need a queue worker if you use `->queue()` or if you have implemented the `ShouldQueue` interface.

Reply:
thats what i meant regarding `ShouldQueue`, for big projects u might forget, so the new note is here to remind you.

Reply:
Yes, I understand the intent, but I think we should explain better or it will lead to more confusion.

Reply:
np, tell me what u think is better and i will add it. 👍

Reply:
What do you think about:

```
## Note about queuing
If you are storing the export using `Excel::queue()` or using the `ShouldQueue` interface, make sure to have a look at the [queuing docs](https://docs.laravel-excel.com/3.1/exports/queued.html)
```

Reply:
perfect, i will update the pr.